README.md

```diff
@@ -16,6 +15,9 @@ Slinky uses Vote Extensions to create an hyperperformant, extremely secure mecha
 many of the highest-performance decentralized applications today. If you would like to integrate Slinky to power your use case, please contact us on our
 [discord](https://discord.gg/PeBGE9jrbu).
 
+> [!NOTE]
+> Slinky is **business-licensed software** under BSL, meaning it requires a license to use or reference. It is source viewable, but [**reach out to us on Discord**](https://skip.money/discord) if you are interested in integrating! We are limiting the number of chains we work with to seven in 2024. We apologize if we run out of capacity.
```

Comment:
do we want to be so explicit with the number of chains we are limiting to to 7?
Could we be a bit less defined?